File: OEM/Domain/IncidentTypes/IncidentTypeDto.cs
using System;

namespace OEMAPI.Domain.IncidentTypes
{
    public class IncidentTypeDto
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Severity { get; set; }
    }
}

File: OEM/src/dto/IncidentTypeDTO.ts
export interface IncidentTypeDTO {
  id: string;
  code: string;
  name: string;
  description: string;
  severity: string;
  parentId?: string | null; 
  parentName?: string;      
}

export interface CreateIncidentTypeDTO {
  code: string;
  name: string;
  description: string;
  severity: string;
  parentId?: string | null; 
}

export interface UpdateIncidentTypeDTO {
  code?: string;
  name?: string;
  description?: string;
  severity?: string;
  parentId?: string | null; 
}
